include/openssl/ssl.h

```diff
@@ -756,41 +756,41 @@ void SSL_CTX_set_next_protos_advertised_cb(SSL_CTX *s,
                                            int (*cb) (SSL *ssl,
                                                       const unsigned char
                                                       **out,
-                                                      unsigned int *outlen,
+                                                      size_t *outlen,
                                                       void *arg), void *arg);
 void SSL_CTX_set_next_proto_select_cb(SSL_CTX *s,
                                       int (*cb) (SSL *ssl,
                                                  unsigned char **out,
-                                                 unsigned char *outlen,
+                                                 size_t *outlen,
                                                  const unsigned char *in,
-                                                 unsigned int inlen,
+                                                 size_t inlen,
                                                  void *arg), void *arg);
 void SSL_get0_next_proto_negotiated(const SSL *s, const unsigned char **data,
                                     unsigned *len);
 # endif
 
-__owur int SSL_select_next_proto(unsigned char **out, unsigned char *outlen,
-                          const unsigned char *in, unsigned int inlen,
-                          const unsigned char *client,
-                          unsigned int client_len);
+__owur int SSL_select_next_proto(unsigned char **out, size_t *outlen,
```

Comment:
I think you missed my comment, sorry. We're `size_t`-ifying internally but shouldn't be changing external APIs. Please revert these, sorry for the confusion.